src/TeamService.ts

```diff
@@ -103,7 +109,15 @@ class TeamService extends Base {
     const { apiEndpointAddress, fetch } = this.lensPlatformClient;
     const url = `${apiEndpointAddress}/teams/${id}/users/${username}`;
 
-    const json = await fetch.delete(url);
+    const json = await throwExpected(
+      async () => fetch.delete(url), {
+        403: () => new ForbiddenException(),
+        404: e =>
+          e?.body.message?.includes("Space not found") ?
+            new SpaceNotFoundException("Space") :
```

Comment:
Isn't that supposed to be the space name?

Reply:
Yes, but we don't know the name. Good point though, I adjusted this so that the name is optional.